docs/diagrams/AddBooleanCommand.puml

```diff
@@ -0,0 +1,40 @@
+
+@startuml
```

Comment:
Please fix the activations, they should all be after the arrow `->`.

Reply:
Okay sure